src/main.rs

```diff
@@ -66,7 +69,10 @@ fn main() {
                 }
 
                 let m = match generate_move(&game, your_color) {
-                    Some(m) => m,
+                    Some(m) => {
+                        println!("{}", m);
```

Comment:
ditto from above

Reply:
Same comment as above, program needs to output move to stdin